### PR TITLE
Widget Bugfix

### PR DESCRIPTION
**Pull request by LouisLouiseKay**

### CODE REVIEW

File: client/src/components/common/SmallTileDetails.tsx [hunk 1]
```diff
@@ -24,6 +24,14 @@ export const SmallTileDetails: React.FC<{
             text-overflow: ellipsis;
             white-space: wrap;
           }
+
+          @media screen and (max-width: ${bp.small}px) {
+            > div {
```
Comment (simonv3) [inline client/src/components/common/SmallTileDetails.tsx:29]: Shouldn't you just define this on the div on the next line?

Comment (LouisLouiseKay) [reply client/src/components/common/SmallTileDetails.tsx:29]: No cause it's for all 3

Comment (simonv3) [reply client/src/components/common/SmallTileDetails.tsx:29]: Then it should be styled component :P

**Comment by simonv3** [comment]
There's a merge conflict

File: client/src/components/common/SmallTileDetails.tsx (hunk 1 above)
Comment (LouisLouiseKay) [reply client/src/components/common/SmallTileDetails.tsx]: Cleaned up !

File: client/src/styles.ts [hunk 2]
```diff
@@ -108,7 +108,7 @@ h1 {
 
 h2 {
   font-size: 1.5rem;
-  line-height: 1.8rem;
+  line-height: 1.5;
```
Comment (simonv3) [inline client/src/styles.ts:111]: This looks like maybe it's undoing previous commits?

File: client/src/translation/en.json [hunk 3]
```diff
@@ -412,7 +412,7 @@
     "questions.howDoesMirloWork.question": "How does Mirlo work?",
     "questions.howDoesMirloWork.answer": "Mirlo allows musicians to set up pages to host their music and set up tiers of monthly support. Fans can then purchase those albums or support musicians through monthly payments and get updates from them when they post on Mirlo.",
     "questions.howMuchDoesMirloCost.question": "How much does Mirlo cost?",
-    "questions.howMuchDoesMirloCost.answer": "We’re exploring a number of funding structures. We’re aiming for musicians to get as much of the price of their albums and subscriptions as possible while still supporting the workers and infrastructure of the co-op. We will likely add ways for purchasers to support the platform directly.",
```
Comment (simonv3) [inline client/src/translation/en.json:415]: Yeah this is reverting a fix I had put in place

**Pull request closed without merging**